curriculum/challenges/english/25-front-end-development/lecture-working-with-regular-expressions/6733aad43b3ebff588a26fb5.md

```diff
@@ -1,14 +1,140 @@
 ---
 id: 6733aad43b3ebff588a26fb5
 title: What Are Regular Expressions, and What Are Some Common Methods?
```

Comment:
finished editing for this one. need to check the video before uploading to youtube. We can merge this in for now. But there will need to be another PR to add in the id.